_data/toc.yaml

```diff
@@ -45,6 +45,7 @@ about:
   - title: What You Need to Know
     path: /wAbout/What-You-Need-to-Know.html
 - sectiontitle: Installing Interject
+  path: /wAbout/SingleUser.html
   section:
   - title: Installing - Single User
     path: /wAbout/SingleUser.html
```

Comment:
do we need to have this item in the drop-down if it's also linked in the item header? Seems redundant.